app/scripts/views/components/barChart/barChart.js

```diff
@@ -89,14 +105,19 @@
       if (logScale) {
         chartInst_.x(d3.scale.log().nice()
           .domain([0.7, opts_.maxDomain]));
+      } else if (data_.smallDataFlag) {
```

Comment:
I'm afraid not. Line 115-120 uses linear() scale, but scientific numbers use log() scale.